src/pages/resultsView/enrichments/MiniFrequencyScatterChart.tsx

```diff
@@ -0,0 +1,219 @@
+        const leastSignificantQ = 0.2;
+        const leastSignificantSize = 1;
+        const mostSignificantSize = 5;
+        const mostSignificantQ = 0.01;
+
+        // coefficients derived to fit f(x) = ae^bx so that f(least significant Q) = least significant size, f(most significant Q) = most significant size
+        const a = leastSignificantSize;
+        const b = (Math.log(mostSignificantSize) - Math.log(leastSignificantSize)) / (mostSignificantQ - leastSignificantQ);
+
+        return (d:IMiniFrequencyScatterChartData)=>{
+            // first clamp the input
+            let q = d.qValue;
+            if (q > leastSignificantQ) {
+                q = leastSignificantQ;
+            } else if (q < mostSignificantQ) {
+                q = mostSignificantQ;
+            }
+
+            return a * Math.exp(b * (q - leastSignificantQ));
+        };
+    }*/
+
+    @computed get splitData() {
+        const significant = [];
+        const insignificant = [];
+        for (const d of this.props.data) {
+            if (d.qValue < 0.05) {
+                significant.push(d);
+            } else {
+                insignificant.push(d);
+            }
+        }
+        return { significant, insignificant };
+    }
+
+    @computed get plotDomainMax() {
+        let maxX = 0;
+        let maxY = 0;
+        for (const d of this.props.data) {
+            maxX = Math.max(maxX, d.x);
+            maxY = Math.max(maxY, d.y);
+        }
+        return Math.max(maxX, maxY);
+    }
+
+    public render() {
+
+        const events = [{
+            target: "data",
+            eventHandlers: {
+                onMouseOver: () => {
+                    return [
+                        {
+                            target: "data",
+                            mutation: (props: any) => {
+                                this.tooltipModel = props;
+                                return {
+                                    datum: Object.assign({}, props.datum, {hovered: true})
+                                };
+                            }
+                        }
+                    ];
+                },
+                onMouseOut: () => {
+                    return [
+                        {
+                            target: "data",
+                            mutation: (props: any) => {
+                                this.tooltipModel = null;
+                                return {
+                                    datum: Object.assign({}, props.datum, {hovered: false})
+                                };
+                            }
+                        }
+                    ];
+                }
+            }
+        }];
+
+        return (
+            <div className="posRelative">
+                <div className="borderedChart inlineBlock" style={{position:"relative"}}>
+                    <VictoryChart containerComponent={<VictorySelectionContainer containerRef={this.svgRef}
```

Comment:
i think we should make these attributes referesnces to methods or variables that hold the components.  kind of a lot of nesting.